src/pages/Sketch/components/Canvas.js

```diff
@@ -251,63 +88,15 @@ function Canvas() {
         link.click();
     }
 
-    function clear() {
-        context.clearRect(0, 0, 700, 500);
-    }
-
     return (
         <>
-            <Toolbox
-                color={color}
-                setColor={setColor}
-                width={width}
-                setWidth={setWidth}
-                opacity={opacity}
-                setOpacity={setOpacity}
-                stroke={stroke}
-                setStroke={setStroke}
-                fill={fill}
-                setFill={setFill}
-            />
             <div className={styles.options}>
-                <div className={styles.shapes}>
-                    <div className={`${styles.shape_box} ${type === 'pen' && styles.active_shape_box}`}
-                    onClick={() => setType("pen")}>
-                        <FaPencilAlt/>
-                    </div>
-                    <div className={`${styles.shape_box} ${type === 'line' && styles.active_shape_box}`}
-                    onClick={() => setType("line")}>
-                        <FaSlash/>
-                    </div>
-                    <div className={`${styles.shape_box} ${type === 'square' && styles.active_shape_box}`}
-                    onClick={() => setType("square")}>
-                        <FaRegSquare/>
```

Comment:
This is not assigned you. Please do not change this

Reply:
Hi bro, Please install the dom-to-pdf dependency to work.

Thank you

Reply:
How is using dom-to-pdf related to the sketch page at all?